_posts/2022-05-23-intellij-quarkus-tools-1.11.0.adoc

```diff
@@ -0,0 +1,124 @@
+
+We are very pleased to announce the 1.11.0 release of Quarkus Tools for IntelliJ.
+This release adds support for Qute templates but also align with LSP4MP 0.4.0 and quarkus-ls 0.11.1.
+
+== Qute templates
+
+When a Quarkus project has Qute in the classpath, editing a Qute template (HTML / JSON / YAML / Text) will add new features into the editor.
+
+Let's do a little tour.
+
+=== Integration between Qute templates and Java
+
+Now any `Template` attribute will show the option to generate a corresponding Qute template, or navigate to the existing template file.
+
+
+image::quarkus-tools1.gif[]
+
+All Qute files paired with a corresponding Java resource supports a number of language client features, such as go to definition, hover, autocompletion, etc.. Each of these we will explore further in the article
+
+image::quarkus-tools2.gif[]
+
+=== Method Parameter Support in Qute Templates
+
+The Qute language server has support for built-in and user-defined Java classes, including parameter completion and validation for their methods. Take advantage of this to easily avoid Java related mistakes.
+
+image::quarkus-tools3.gif[]
+
+=== Java Type Hover Support for Qute Templates
+
+Alongside support for Java completion and validation, the extension also supports type hovering for built-in and user-defined Java classes.
+
+image::quarkus-tools4.gif[]
+
+=== Qute Section Completion
+
+Qute templates use sections in its templating language to power any logic required when compiling and displaying the resulting document. To support this, the extension’s Qute support populates a completion list with all available Qute sections, along with any parameters they take and a small code snippet of its usage.
+
+image::quarkus-tools5.gif[]
+
+=== Support for Qute Validation
+
+Java and Qute validation is included for Qute templates with Quarkus Tools for Visual Studio Code, including diagnostics.
+
+image::quarkus-tools6.gif[]
+
+=== Completion for Qute Value Resolvers
+
+Paired with corresponding Java methods, built-in Java classes also have built-in Qute value resolvers made available by the Qute templating engine that is evaluated and outputs, or resolves to, a value. Quarkus Tools for Visual Studio Code supports completion, hover and snippet generation for these built-in value resolvers.
+
+image::quarkus-tools7.gif[]
+
+=== Support for @TemplateExtension Methods
+
+Template extension methods are used to extend the set of accessible properties of data objects. Qute support in Quarkus Tools for Visual Studio Code has support for marked `@TemplateExtension` Java methods, which extend a given data model.
+
+image::quarkus-tools8.gif[]
+
+=== Support for Directly Injecting Beans into Qute Templates
+
+Support for Directly Injecting Beans into Qute Templates
+
+A CDI bean annotated with `@Named` can be referenced in any template through cdi and/or inject namespaces. Quarkus Tools for Visual Studio Code provides completion and definition navigation support for Beans.
```

Comment:
The news is about Intellij and you're talking about Visual Studio Code. Is it expected?